frontend/src/scenes/insights/insightLogic.ts

```diff
@@ -288,133 +258,6 @@ export const insightLogic = kea<insightLogicType>([
-                        if (
-                            values.savedInsight?.id &&
-                            objectsEqual(cleanFilters(filters), cleanFilters(values.savedInsight.filters ?? {}))
-                        ) {
-                            // Instead of making a search for filters, reload the insight via its id if possible.
-                            // This makes sure we update the insight's cache key if we get new default filters.
-                            apiUrl = combineUrl(`api/projects/${currentTeamId}/insights/${values.savedInsight.id}`, {
-                                refresh: true,
-                            }).url
-                            fetchResponse = await api.getResponse(apiUrl, methodOptions)
-                        } else {
-                            const params = {
-                                ...filters,
-                                ...(refresh ? { refresh: true } : {}),
-                                client_query_id: queryId,
-                                session_id: currentSessionId(),
-                            }
-                            ;[fetchResponse, apiUrl] = await legacyInsightQuery({
-                                filters: params,
-                                currentTeamId,
-                                methodOptions,
-                                refresh,
-                            })
-                        }
-                        response = await getJSONOrThrow(fetchResponse)
-                    } catch (e: any) {
-                        if (shouldCancelQuery(e)) {
-                            actions.abortQuery({
-                                queryId,
-                                view: insight,
-                                scene: scene,
-                                exception: e,
-                            })
-                        }
-                        breakpoint()
-                        actions.endQuery({
-                            queryId,
-                            view: insight,
-                            scene: scene,
-                            lastRefresh: null,
-                            nextAllowedRefresh: null,
-                            exception: e,
-                        })
-                        if (dashboardItemId && dashboardsModel.isMounted()) {
-                            dashboardsModel.actions.updateDashboardRefreshStatus(dashboardItemId, false, null)
-                        }
-                        if (isFunnelsFilter(filters)) {
-                            eventUsageLogic.actions.reportFunnelCalculated(
-                                filters.events?.length || 0,
-                                filters.actions?.length || 0,
-                                filters.interval || '',
-                                filters.funnel_viz_type,
-                                false,
-                                e.message
-                            )
```

Comment:
Are all reporting actions like these also called in the new system? We might have different insights/dashboards dependent on them... Not sure.

Reply:
I remember implementing some of them for the new system, but will double check. Wouldn't put too much trust into these anyway, since I don't think all insight state and caching options are properly handled (or even hard to say what we want to know - e.g. should events on experiments count).

This event in particular doesn't seem to be used anywhere though. 30-day usage is empty and a [quick query against metabase](http://metabase-prod-us/question/165-funnel-result-calculated-usage) shows that the event was used only in ad-hoc insights and tmp actions, last 2021.

Therefore I'm opting to remove the event completely in favor of a simpler codebase. The same insights can be made from the "insight viewed" event anyway.